examples/dendrogram.py

```diff
@@ -146,8 +148,7 @@ def _dendrogram(neuron_, show_diameters=True):
     '''Main dendrogram function
     '''
     from copy import deepcopy
-    import sys
-    sys.setrecursionlimit(3000)
+
     # neuron is copied because otherwise the tree modifications that follow
     # will be applied to our original object
     neuron = deepcopy(neuron_)
```

Comment:
Yep, I agree. I also do not like the recursive nature of the dendrogram creation and I will probably make it iterative at some point if possible but it will need a significant amount of time.

Can we leave the recursion limit like this for the time being (I think it is probably needed by the dendrogram as well) and for example first implement the copying of the neuron in the core folder?